modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/restapi/admin/throttlingpolicy/ThrottlePolicyExportImportTestCase.java

```diff
@@ -0,0 +1,597 @@
+    private final String APIM_VERSION = "v4.1.0";
+    private final String description = "This is a test throttle policy";
+    private final String advancedPolicyName = "TestPolicyAdvanced";
+    private final String applicationPolicyName = "TestPolicyApplication";
+    private final String customPolicyName = "TestPolicyCustom";
+    private final String subscriptionPolicyName = "TestPolicySubscription";
+    private final String THROTTLE_POLICY_TYPE = "throttling policy";
+    private final String ADVANCED_POLICY_SUBTYPE = "advanced policy";
+    private final String APPLICATION_POLICY_SUBTYPE = "application policy";
+    private final String SUBSCRIPTION_POLICY_SUBTYPE = "subscription policy";
+    private final String CUSTOM_POLICY_SUBTYPE = "custom rule";
+    private final String advancedPolicyType = "api";
+    private final String applicationPolicyType = "app";
+    private final String subscriptionPolicyType = "sub";
+    private final String customPolicyType = "global";
+    private final String exportRestAPIResource = "/throttling/policies/export";
+    private final String importRestAPIResource = "/throttling/policies/import";
+    private AdvancedThrottlePolicyDTO AdvancedPolicyDTO;
+    private ApplicationThrottlePolicyDTO ApplicationPolicyDTO;
+    private CustomRuleDTO CustomPolicyDTO;
+    private SubscriptionThrottlePolicyDTO SubscriptionPolicyDTO;
+    private AdminApiTestHelper adminApiTestHelper;
+    private String exportUrl;
+    private String importUrl;
+    private String advancedPolicyId;
+    private String applicationPolicyId;
+    private String subscriptionPolicyId;
+    private String customPolicyId;
+    private File exportedFileAdvancedPolicy;
+    private File exportedFileApplicationPolicy;
+    private File exportedFileCustomPolicy;
+    private File exportedFileSubscriptionPolicy;
+
+    @Factory(dataProvider = "userModeDataProvider")
+    public ThrottlePolicyExportImportTestCase(TestUserMode userMode) {
+        this.userMode = userMode;
+    }
+
+    @DataProvider public static Object[][] userModeDataProvider() {
+        return new Object[][] { new Object[] { TestUserMode.SUPER_TENANT_ADMIN },
+                new Object[] { TestUserMode.TENANT_ADMIN } };
+    }
+
+    @BeforeClass(alwaysRun = true)
+    public void setEnvironment() throws Exception {
+        super.init(userMode);
+        adminApiTestHelper = new AdminApiTestHelper();
+
+        exportUrl = adminURLHttps + APIMIntegrationConstants.REST_API_ADMIN_CONTEXT_FULL_0 + exportRestAPIResource;
+        importUrl = adminURLHttps + APIMIntegrationConstants.REST_API_ADMIN_CONTEXT_FULL_0 + importRestAPIResource;
+
+        addAdvancedPolicy();
+        addApplicationPolicy();
+        addSubscriptionPolicy();
+        addCustomPolicy();
+    }
+
+    /**
+     * Adds a new subscription policy
+     */
+    public void addSubscriptionPolicy() throws Exception {
+
+        String internal_creator = "Internal/creator";
```

Comment:
Check if there is a constant already defined for this somewhere else

Reply:
Fixed with 3106d801961d2d8e82a5a35117427b8337a92455